source/Files.cpp

```diff
@@ -296,6 +296,12 @@ time_t Files::Timestamp(const filesystem::path &filePath)
 
 void Files::Copy(const filesystem::path &from, const filesystem::path &to)
 {
+#ifdef WIN32
```

Comment:
Is there any difference between `WIN32` and `_WIN32`?

Reply:
IIRC there isn't these days, but _WIN32 used to be more generally defined.

```suggestion
#ifdef _WIN32
```